deploy/ncsa/docker-compose.yml

```diff
@@ -26,35 +53,181 @@ services:
     ipc: host
     pid: host
     networks:
-      dds-network:
-        ipv4_address: "141.142.238.101"
       love:
     depends_on:
       - nginx
     volumes:
       - /tmp/docker_tmp/:/tmp/
       - ${OSPL_CONFIG_PATH}:${OSPL_MOUNT_POINT}
   #----- LOVE PRODUCER --------------
-  producer:
-    <<: *service
-    container_name: love-producer
-    image: ts-dockerhub.lsst.org/love-producer:${cycle}
+  Watcher:
+    <<: *base-producer
+    container_name: producer-watcher-image-mount
```

Comment:
Why are these image mounts? Are they doing something different/special than the other producers? Same goes for the others named this way.

Reply:
Fixed the names.